--- h5augjpss-1.0.0/CMakeLists.txt.orig	2011-06-23 19:48:48.000000000 +0200
+++ h5augjpss-1.0.0/CMakeLists.txt	2013-04-02 21:16:47.745819501 +0200
@@ -257,13 +257,13 @@
 # Define Sources
 #-----------------------------------------------------------------------------
 SET (h5augjpss_SRCS
-    h5aj_main.c
-    h5aj_command.c
-    h5aj_read_xml.c
-    h5aj_writenc4.c
-    h5aj_restore.c
-    h5augjpss.h
-    h5aj_private.h
+    src/h5aj_main.c
+    src/h5aj_command.c
+    src/h5aj_read_xml.c
+    src/h5aj_writenc4.c
+    src/h5aj_restore.c
+    src/h5augjpss.h
+    src/h5aj_private.h
 )
 
 ADD_EXECUTABLE (h5augjpss ${h5augjpss_SRCS})
